packages/esm-tb-app/src/config-schema.ts

```diff
@@ -37,15 +37,20 @@ export const configSchema = {
       monthOfTreatment: '1418AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
       adherenceAssessment: '164075AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
       nextAppointmentDate: '5096AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      DateEnrolledToTB: '161552AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      TreatmentId: '164415AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      TreatmentStartDate: '164416AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      Drregimen: '16fd7307-0b26-4c8b-afa3-8362baff4042',
```

Comment:
There's already a conept `regimen` with the same concept so i suggest to delete this and use the existing

Reply:
Done